app/src/main/res/layout/continue_playing_fragment.xml

```diff
@@ -2,18 +2,26 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
 
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <data>
+
+    <import type="org.oppia.app.R" />
```

Comment:
Doesn't seem to be needed--please remove.

Reply:
Removed.